OpenRA.Mods.Common/Effects/RepairIndicator.cs

```diff
@@ -50,13 +49,16 @@ public IEnumerable<IRenderable> Render(WorldRenderer wr)
 			if (building.Destroyed || wr.World.FogObscures(building) || rb.Repairers.Count == 0)
 				return SpriteRenderable.None;
 
-			var palette = wr.Palette(palettePrefix + rb.Repairers[shownPlayer % rb.Repairers.Count].InternalName);
+			var palette = wr.Palette(rb.Info.IndicatorPalettePrefix + rb.Repairers[shownPlayer % rb.Repairers.Count].InternalName);
```

Comment:
This may crash if a mod doesn't define a player-specific repair palette.  You will need to do the IndicatorPalette check *before* doing any palette lookups.